src/main/scala/ReleaseExtra.scala

```diff
@@ -225,7 +225,7 @@ object ReleaseStateTransformations {
   }
 
   lazy val publishArtifacts = ReleaseStep(
-    action = publishArtifactsAction,
+    action = runPublishArtifactsAction,
```

Comment:
Yep.  There's a few solutions - the simplest is we could add the `publishArtifactsAction` setting to the plugin project settings, which means it will be automatically set for every project.  We could also make the action a setting for a key, rather than a task - this will be a breaking change from 0.8.5 to 0.8.6, but maybe that's necessary.